circle.yml

```diff
@@ -131,13 +132,55 @@ test:
     - while [ ! -e ~/screenshots_are_ready ]; do sleep 1; done:
         timeout: 300
 
+    # Stitch together images
+    # TODO: Split this into multiple yaml lines. I (Michael)
+    #       niavely tried to split this into mutiple yaml lines
+    #       but was unsucessful in doing do.
     - |-
-      dir=$CIRCLE_ARTIFACTS/imgs
-      for x in $(ls $dir)
+      img_dir=$CIRCLE_ARTIFACTS/imgs/
+      for x in $(ls $img_dir)
       do
-        convert $dir/$x/*.png -append $dir/$x.png
+        convert $img_dir/$x/*.png -append $img_dir/$x.png
       done
 
+      # Remove all directories in $CIRCLE_ARTIFACTS/img
+      # Currently the pieces aren't kept around. If it's
+      # desirable to keep them around, we should use
+      #    cp -r $dir $CIRCLE_ARTIFACTS/img_pieces
+      # The reason the image pieces aren't currently kept
+      # around is that it was leading to a problem. Specifically,
+      # when we get the previous images, we niavely grab any *.png,
+      # including the pieces images. This compounded so that each
+      # iteration of a test run would have all of the images from
+      # the previous test run plus whichever new images were generated.
+      rm -R -- $img_dir/*/
+
+      # Install utility we need
+      npm install -g json
+
+      # Download the latest mathquill artifacts.
+      curl $(curl https://circleci.com/api/v1/project/mathquill/mathquill/latest/artifacts \
```

Comment:
Master was intended. Forgot to do that. I'll fix that now. Unless you think it would be preferable to leave things they way they are now.